sapcc/oslo.vmware
# Add PBM WSDL for VSphere 8.0 - wallaby-m3

- Updated from global requirements
- Updated from global requirements
- Updated from global requirements
- Updated from global requirements
- Make sure host in maintenance mode excluded from image upload
- Wrong member variable name in class Datastore
- Updated from global requirements
- Updated from global requirements
- Do not prune some special XML elements which are empty
- Updated from global requirements
- Add hacking rule to prevent log translations
- Improving cover testenv
- Use request_handler() for ExtensionManager APIs
- Enable warning-is-error in doc build
- Switch from oslosphinx to openstackdocstheme
- rearrange existing documentation to fit the new standard layout
- Update URLs in documents according to document migration
- Updated from global requirements
- Update reno for stable/pike
- Updated from global requirements
- Imported Translations from Zanata
- Updated from global requirements
- Updated from global requirements
- Imported Translations from Zanata
- Updated from global requirements
- Updated from global requirements
- Remove setting of version/release from releasenotes
- Avoid tox_install.sh for constraints support
- Remove -U from pip install
- Fix the repeating titles
- Updated from global requirements
- Cleanup test-requirements
- Updated from global requirements
- Updated from global requirements
- Updated from global requirements
- Update reno for stable/queens
- Imported Translations from Zanata
- Imported Translations from Zanata
- Updated from global requirements
- Update links in README
- Updated from global requirements
- Updated from global requirements
- pypy is not checked at gate
- add lower-constraints job
- Updated from global requirements
- fix lower constraints and uncap eventlet
- set default python to python3
- Trivial: Update pypi url to new url
- fix a typo:s/extention/extension
- Remove stale pip-missing-reqs tox test
- PBM WSDL for vSphere 6.7
- Refactor _start_transfer
- Fix session re-establish failure with PBM APIs
- Specify CA store when making secure connections with urllib3
- fix tox python3 overrides
- Fix docstring which breaks doc generation
- Add traversal spec for datastoreFolder
- Add release notes link to README
- Switch to stestr
- Add utility function for datastore clusters
- Add function for getting recommended datastore
- Utility method to check SDRS enabled status
- Update reno for stable/rocky
- py37: deal with Exception repr changes
- import zuul job settings from project-config
- Set request context when polling vCenter tasks
- add python 3.6 unit test job
- add lib-forward-testing-python3 test job
- Imported Translations from Zanata
- Check if there is a context set before updating the store
- Add DVS utility methods
- Clean up .gitignore references to personal tools
- delete useless indent
- Change openstack-dev to openstack-discuss
- Use template for lower-constraints
- Change python3.5 job to python3.7 job on Stein+
- add python 3.7 unit test job
- SDRS recommendation for create VM
- Return None if no suitable datastore is found
- Update master for stable/stein
- Fix doc build error under Python 3
- Fix invalid escape sequence warning
- When looping call fails do not log the exception trace
- OpenDev Migration Patch
- Replace git.openstack.org URLs with opendev.org URLs
- Add nosec to skip bandit check
- Update NFC lease progress
- Set locale to english
- Ensure that log message with non-ascii code does not raise exception
- Dropping the py35 testing
- Sync Sphinx requirement
- Add local bindep.txt
- Add Python 3 Train unit tests
- Follow the new PTI for document build
- Update master for stable/train
- tox: Keeping going with docs
- Switch to Ussuri jobs
- tox: Trivial cleanup
- Pass "store_id" to backend when transfer volume to image
- [ussuri][goal] Drop python 2.7 support and testing
- Add ability to change read chunk size
- Adds standard file operation functions
- reword releasenote for py27 support dropping
- Ensure lease polling raises proper exception
- remove outdated header
- Drop use of six
- Pass "base_image_ref" to backend when transfer volume to image
- Update the tox minversion parameter.
- Fix object has no attribute 'readinto' in Python3
- Update hacking for Python3
- Document disable flake8/hacking warnings
- Use unittest.mock instead of third party mock
- Update master for stable/ussuri
- Add Python3 victoria unit tests
- Add release notes links to doc index
- Add py38 package metadata
- Bump default tox env from py37 to py38
- Align contributing doc with oslo's policy
- Remove translation sections from setup.cfg
- Remove the unused coding style modules
- Switch to newer openstackdocstheme and reno versions
- Imported Translations from Zanata
- Fix hacking min version to 3.0.1
- Validation of VMDK upload completion
- Add moref helper functions
- Add backend-independent access to cookiejar
- Bump bandit version
- [goal] Migrate testing to ubuntu focal
- Adding pre-commit
- Update master for stable/victoria
- Add Python3 wallaby unit tests
- Imported Translations from Zanata
- Add PBM WSDL for vSphere 7.0
- Move flake8 as a pre-commit local target.
- Update .gitreview for stable/wallaby
- Update TOX_CONSTRAINTS_FILE for stable/wallaby
- [stable-only] Pin virtualenv
- Add monkey-patching of suds to oslo.vmware
- Adding tell,seek,flush,fileno methods in FileHandle
- Add is_vim_instance() helper function
- Add serialize_object() helper function
- Ignore already deleted lease
- Replace suds-jurko with suds-community
- Pull images from Swift URL
- Return and close connection for read_connection
- Add FileReadHandle
- Add common headers to _create_connection
- Accept pre-build cookie for FileRead/WriteHandle
- DatastoreUrl as parameter for FileRead/WriteHandle
- Add .sleep() to avoid orphaned VMDKs
- VmdkReadHandle: gracefully shutdown and close the socket
- Monkeypatch suds for json output
- Added Deserialize and compatible Serialize
- [SAP] Added set_size() to rw_handles
- UrlPullHandle: fix ssl_thumbprint() to return a plain string
- Add PBM WSDL for vSphere 8.0
